content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+<div class="alert alert-info">
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+Profiling libraries are shipped within the following tracing language library. Refer to the dedicated language page to learn how to enable profile connections for your application:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Profile explorer
+
+Once your profile collection is set up, profiles are available in the [APM > Profiling][1] page within Datadog:
+
+{{< img src="tracing/profiling/profiling_main_page.png" alt="Profiling main page">}}
+
+Each row is a profile of a process for a short amount of time. Depending on the language, these processes are profiled between 15s and 60s.
```

Comment:
Updated suggestion-
> Each row is a profile of a process for a short amount of time. By default profiles are uploaded once a minute.

The fact that they're uploaded once a minute may be more important than the duration of the profiles in understanding the search page. And the duration is actually more complicated than this and is shown in each profile.